jfrog_platform_cloud/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "798102cb-6c52-4a16-bc1b-48c2e6b54e71",
+  "app_id": "jfrog-platform-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze JFrog Artifactory Cloud logs",
+    "title": "JFrog Platform Cloud",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard, HTTP requests",
+        "image_url": "images/jfrog_platform_cloud_logs_0.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard, Request logs",
+        "image_url": "images/jfrog_platform_cloud_logs_1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard, Operations",
+        "image_url": "images/jfrog_platform_cloud_logs_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Log Collection",
```

Comment:
Are any other categories relevant to this integration? You can find the list here: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags

Reply:
Modified to better reflect the integration purpose.